packages/core/src/common/command.ts

```diff
@@ -33,6 +33,7 @@ export interface Command {
      * A label of this command.
      */
     label?: string;
+    originalLabel?: string;
```

Comment:
It might be useful to assist the transition to this system to create a utility that will generate this structure from commands that adhere to the older interface? Something like 
```typescript
export function toLocalizedCommand(command: Command, nlsLabelKey: string = command.id, nlsCategoryKey: string | undefined = command.category): Command {
    return {
        ...command,
        label: command.label && nls.localize(nlsLabelKey, command.label),
        originalLabel: command.label,
        category: nlsCategoryKey && command.category && nls.localize(nlsCategoryKey, command.category),
        originalCategory: command.category,
    };
}
```

Reply:
I like this, I added it. You can see its usage in the `workspace-commands.ts`.